### ISSUE TITLE
Build errors when building for platforms other than Android or iOS.

### ISSUE THREAD

**Issue by joejo-unity** [googlevr/cardboard#55]
When switching to other platforms in the Build Settings in Unity, then trying to build, you get a number of errors due to DllImport attribute definitions using items that don't exist. I may want to create a cross platform application that I wish to target for Standalone or Lumin or something else as well as Cardboard on Android and iOS. I know functionality from Cardboard won't be available but I don't expect my project to error out.

in Api.cs you may want to change this:

```
       [DllImport(ApiConstants.CardboardApi)]
        private static extern void CardboardQrCode_scanQrCodeAndSaveDeviceParams();

        [DllImport(ApiConstants.CardboardApi)]
        private static extern void CardboardQrCode_getSavedDeviceParams(
          out IntPtr encodedDeviceParams, out int size);

        [DllImport(ApiConstants.CardboardApi)]
        private static extern void CardboardQrCode_destroy(IntPtr encodedDeviceParams);

        [DllImport(ApiConstants.CardboardApi)]
        private static extern int CardboardQrCode_getQrCodeScanCount();

        [DllImport(ApiConstants.CardboardXRUnity)]
        private static extern void CardboardUnity_setDeviceParametersChanged();
```

to something like this: 

```
        #if UNITY_ANDROID || UNITY_IOS
        [DllImport(ApiConstants.CardboardApi)]
        private static extern void CardboardQrCode_scanQrCodeAndSaveDeviceParams();

        [DllImport(ApiConstants.CardboardApi)]
        private static extern void CardboardQrCode_getSavedDeviceParams(
          out IntPtr encodedDeviceParams, out int size);

        [DllImport(ApiConstants.CardboardApi)]
        private static extern void CardboardQrCode_destroy(IntPtr encodedDeviceParams);

        [DllImport(ApiConstants.CardboardApi)]
        private static extern int CardboardQrCode_getQrCodeScanCount();

        [DllImport(ApiConstants.CardboardXRUnity)]
        private static extern void CardboardUnity_setDeviceParametersChanged();
#else
        private static void CardboardQrCode_scanQrCodeAndSaveDeviceParams() { }

        private static void CardboardQrCode_getSavedDeviceParams(out IntPtr encodedDeviceParams, out int size) 
        {
            encodedDeviceParams = IntPtr.Zero;
            size = 0;
        }

        private static void CardboardQrCode_destroy(IntPtr encodedDeviceParams) { }

        private static int CardboardQrCode_getQrCodeScanCount() { return 0;  }

        private static void CardboardUnity_setDeviceParametersChanged() { }
#endif
```

same with XRLoader.cs:
This: 

```
        [DllImport(ApiConstants.CardboardXRUnity)]
        private static extern void CardboardUnity_setScreenParams(
            int x, int y, int width, int height);

        [DllImport(ApiConstants.CardboardXRUnity)]
        private static extern void CardboardUnity_setWidgetCount(int count);

        [DllImport(ApiConstants.CardboardXRUnity)]
        private static extern void CardboardUnity_setWidgetParams(
            int i, IntPtr texture, int x, int y, int width, int height);

        [DllImport(ApiConstants.UnityCardboardJni)]
        private static extern void CardboardUnity_initializeAndroid(IntPtr context);
```

to: 

```
#if UNITY_ANDROID || UNITY_IOS
        [DllImport(ApiConstants.CardboardXRUnity)]
        private static extern void CardboardUnity_setScreenParams(
            int x, int y, int width, int height);

        [DllImport(ApiConstants.CardboardXRUnity)]
        private static extern void CardboardUnity_setWidgetCount(int count);

        [DllImport(ApiConstants.CardboardXRUnity)]
        private static extern void CardboardUnity_setWidgetParams(
            int i, IntPtr texture, int x, int y, int width, int height);

        [DllImport(ApiConstants.UnityCardboardJni)]
        private static extern void CardboardUnity_initializeAndroid(IntPtr context);
#else
        private static void CardboardUnity_setScreenParams(int x, int y, int width, int height) { }

        private static void CardboardUnity_setWidgetCount(int count) { }

        private static void CardboardUnity_setWidgetParams(int i, IntPtr texture, int x, int y, int width, int height) { }

        private static void CardboardUnity_initializeAndroid(IntPtr context) { }
#endif
```

**Comment by jballoffet** [comment]
Thanks for the feature request. We will add this to our backlog. However, if you want to send your proposed changes, please read CONTRIBUTING.md and follow the process. Patches are welcome.

**Comment by jballoffet** [comment]
A fix for this has just been submitted and it will be included in the next release.

**Comment by chaosemer** [comment]
This was fixed in v1.3.0.
